src/supa/connection/fsm.py

```diff
@@ -123,5 +198,5 @@ def plot_fsm(fsm: StateMachine, name: str) -> None:
         dg.render(filename=name, directory=output_path, cleanup=True, format="png")
```

Comment:
Are the PNG in this MR a result of running graphviz? (e.g. a build artefact?) : then it might be good to remove them from the git repo.

Reply:
The images were already part of the documentation in the docs folder. They are not a build artefact, I recreated them by hand once after I added a missing state transition in one of the state machines.